xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererDX.cpp

```diff
@@ -248,8 +248,10 @@ COverlayImageDX::COverlayImageDX(CDVDOverlayImage* o)
     else
 #endif
     {
-      /* render aligned to screen to avoid cropping problems */
-      m_align  = ALIGN_SCREEN;
+      if (abs(static_cast<double>(sourceWidth) / sourceHeight - static_cast<double>(o->source_width) / o->source_height) < 0.001)
+        m_align = ALIGN_VIDEO;
```

Comment:
correct is setting it always to ALIGN_VIDEO. the old comment makes zero sense to me.

Reply:
setting it always to ALIGN_VIDEO results is same behaviour we have for SSA